nncore/utils/logger.py

```diff
@@ -24,10 +24,9 @@ def update_figure(self, tag, image, step):
         self.writer.add_figure(tag, image, step)
 
 
-def image_batch_show(batch, ncol=5, fig_size=(30, 10)):
-    # https://stackoverflow.com/questions/51329159/how-can-i-generate-and-display-a-grid-of-images-in-pytorch-with-plt-imshow-and-t
-    grid_img = torchvision.utils.make_grid(batch, nrow=ncol)
-    return grid_img.float()
+def image_batch_show(batch, ncol=5, fig_size=(30, 10), normalize=False):
```

Comment:
what is `fig_size` argument used for?
Moreover, if this does nothing more than calling `torchvision.utils.make_grid`, try define an alias name instead.